modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -119,6 +120,27 @@ def test_read_csv_without_glob(self):
             with pytest.raises(FileNotFoundError):
                 pd.read_csv_glob("s3://nyc-tlc/trip data/yellow_tripdata_2020-")
 
+    @pytest.mark.parametrize(
+        "parse_dates",
+        [pytest.param(value, id=id) for id, value in parse_dates_values_by_id.items()],
+    )
+    def test_read_single_csv_with_parse_dates(self, parse_dates):
+        try:
+            pandas_df = pandas.read_csv(time_parsing_csv_path, parse_dates=parse_dates)
+        except Exception as pandas_exception:
+            with pytest.raises(Exception) as modin_exception:
+                modin_df = pd.read_csv_glob(
```

Comment:
if this is a single CSV, why are we using `read_csv_glob`?

Reply:
This CSV is checking the `parse_dates` functionality, not the "reading from a glob path" functionality. Generally I think it's better for each test case to [test a single thing](https://osherove.com/blog/2005/4/3/a-unit-test-should-test-only-one-thing.html). Here, I think we should test complex `parse_dates` behavior without also testing glob paths. We do the same thing in `test_read_csv_empty_frame` [here](https://github.com/modin-project/modin/blob/f1de99167be9b357a5710f71b320316642121e84/modin/experimental/pandas/test/test_io_exp.py#L112).